README.md

```diff
@@ -42,19 +42,23 @@ Add `svelte-windicss-preprocess` to your `rollup.config.js`.
 
```

Comment:
can you add some description here that make sure people understand that typscript is **optional**